zephyrproject-rtos/zephyr
# tests: http_server: fix failing TLS tests due to moved certificates

Testcase uses certificates from the sample application at samples/net/sockets/http_server. These were modified and moved into a different folder by a previous commit, causing a build failure in the test.

Testcase updated to:
- use new path to certificates
- update available cipher suites, since certificates now use ECDSA
- update expected hostname in certificate

The CA certificate in the sample app is also converted to .der format for inclusion in the test (the content is exactly the same as the PEM version, just converted to DER).
